File: cash/cash.py
from cs50 import get_float
from sys import exit
from math import floor


def main():
    cent = 0
    while(cent <= 0):
        cent = round((get_float("Cash Owed :")) * 100)

    count = 0
    while(cent != 0):
        divider = determiner(cent)
        count += floor(cent/divider)
        cent = cent % divider

    print(count)
    exit(0)


def determiner(cent):
    if((cent / 25) >= 1):
        return 25
    elif(cent / 10 >= 1):
        return 10
    elif(cent / 5 >= 1):
        return 5
    else:
        return 1


main()

File: cash.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>

int determiner(int);

int main(void)
{
    int cent = 0;
    while (cent <= 0)
    {
        cent = round(get_float("Cash Owed : ") * 100);
    }

    int count = 0;

    while (cent != 0)
    {
        int divider = determiner(cent);
        count += cent / divider;
        cent = cent % divider;
    }
    printf("%i\n", count);
}        

int determiner(int cent)
{
    if ((cent / 25) != 0)
    {
        return 25;
    }
    if ((cent / 10) != 0)
    {
        return 10;
    }
    if ((cent / 5) != 0)
    {
        return 5;
    }
    if ((cent / 1) != 0)
    {
        return 1;
    }
    return 0;
}
